src/scss/components/layout/_index.scss

```diff
@@ -132,44 +132,34 @@ body.hxVertical {
 
 // ========== HORIZONTAL LAYOUT ==========
 body.hxHorizontal {
-  // Legacy Grid Implementation
-  display: -ms-grid;
   -ms-grid-columns: 1fr;
   -ms-grid-rows: $header-height 1fr $footer-height;
-
-  // Modern Grid Implementation
-  display: grid;
+  display: -ms-grid; // Legacy Grid (IE)
+  display: grid; // Modern Grid
   grid-template-areas:
-    'header'
-    'app'
-    'footer';
+    "header"
+    "app"
+    "footer";
   grid-template-columns: 1fr;
   grid-template-rows: $header-height 1fr $footer-height;
 
   #head {
-    // Legacy Grid
     -ms-grid-column: 1;
     -ms-grid-row: 1;
-
-    // Modern Grid
     grid-area: header;
   }
 
   #app {
-    overflow: hidden;
-
-    // Legacy Grid
     -ms-grid-column: 1;
     -ms-grid-row: 2;
-
-    // Modern Grid
     grid-area: app;
+    overflow: hidden;
```

Comment:
styles rearranged/refactored to properly alphabetize and satisfy linting configuration